OneCommunityGlobal/HighestGoodNetworkApp
# Vishala Consumables Page View

# Description
This PR is about the Consumables List View from Phase 2 with Sorting based on project name and item name , Filtering based on selected values of project and consumable  and Modal implementation for viewing update and purchase history.
Implements # (WBS)  8.1.1

## Related PRS (if any):
This frontend PR is related to the [#672](https://github.com/OneCommunityGlobal/HGNRest/pull/672) backend PR.

## Main changes explained:
- Created components for Consumables

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. verify all the table functionalities like open/close modals , sorting and filtering

## Screenshots or videos of changes:

[Untitled_ Dec 29,2023 5_14 PM.webm](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/65274029/b86e7b96-4b0a-4572-a4c9-10a059a5400b)
